test/component/Tooltip/Tooltip.visibility.spec.tsx

```diff
@@ -409,6 +409,50 @@ describe('Tooltip visibility', () => {
       expect(tooltip2.getAttribute('style')).toContain(expectedTransform);
     });
 
+    it('should move onTouchMove', async context => {
+      // TODO: these charts currently do not work onTouchMove. Did they before?
```

Comment:
Charts with `item` eventtype will dispatch their events from individual graphical items' event handlers. We might need to add `onTouchMove` there too. Example: https://github.com/recharts/recharts/blob/3.x/src/polar/Pie.tsx#L307

Reply:
Yeah I forgot about that. So not quite done but close then 👍